src/app/view/util/svg.mouse.controller.ts

```diff
@@ -168,6 +168,10 @@ export class SVGMouseController {
   }
 
   moveNetzgrafikEditorViewFocalPoint(center: Vec2D) {
+    if (this.viewboxProperties === undefined) {
+      console.log("moveNetzgrafikEditorViewFocalPoint", null);
```

Comment:
Don't forget to remove `console.log` before merging, if it's for test purpose

Reply:
Fixed - thank you